web/wp-config.php

```diff
@@ -14,180 +14,75 @@
  */
 require_once( $rootPath . '/vendor/autoload.php' );
 
-/*
- * Fetch .env
- */
-if ( ! isset( $_ENV['PANTHEON_ENVIRONMENT'] ) && file_exists( $rootPath . '/.env' ) ) {
-	$dotenv = Dotenv\Dotenv::create($rootPath);
-	$dotenv->load();
-	$dotenv->required( array(
-		'DB_NAME',
-		'DB_USER',
-		'DB_HOST',
-	) )->notEmpty();
-}
-
-/**
- * Disallow on server file edits
- */
-define( 'DISALLOW_FILE_EDIT', true );
-define( 'DISALLOW_FILE_MODS', true );
```

Comment:
So this is the crux of this PR; this also appears to be something picked up from example-wordpress-composer. Is the purpose here to avoid problems on the TEST and LIVE environments? Perhaps we could move these directives to the wp-config-pantheon.php file, and guard them to only be defined on test and live (and if not already defined)?  That would be the best solution. A matching PR in pantheon-systems/wordpress would also be helpful.

Reply:
@greg-1-anderson yes, I believe we're already doing this in this PR (here: https://github.com/pantheon-upstreams/wordpress-project/pull/6/files#diff-0d064dc4acbbc571d7e81ae8c6b43bf48b93d88dafa4c5e7e52fdb17f1661548R73-R83)